src/components/axes/timeAxis.ts

```diff
@@ -1,6 +1,11 @@
 ///<reference path="../../reference.ts" />
 
 module Plottable {
+
+export enum TimeInterval {
+  second, minute, hour, day, week, month, year
```

Comment:
Is it possible to use a "`string`y enum", so that the values are human-readable after compilation?

```Typescript
export module TimeInterval {
  export var second = "second";
  export var minute = "minute";
  ...
}
```